components/form/CustomerForm.tsx

```diff
@@ -29,7 +29,7 @@ const validateCustomerFields = (fields: CustomerFields): Customer => {
     }
 };
 
-function CustomerForm(): JSX.Element {
+function CreateCustomerForm(): JSX.Element {
```

Comment:
Change the filename too?

But this naming convention is better. `CustomerForm` doesn't really tell what the form is for.